src/amp-story-player-impl.js

```diff
@@ -133,7 +134,8 @@ export class AmpStoryPlayer {
 
   /** @public */
   buildCallback() {
-    this.stories_ = toArray(this.element_.querySelectorAll('a'));
+    initLogConstructor();
```

Comment:
Does it need to be executed as many times as we have players? If not, can we move it to another file?

Reply:
Moved it to the manager.